README.md

```diff
@@ -42,6 +42,9 @@ Tutorial 7: Adding Custom Waveforms to PyCBC
 Tutorial 8: Black hole ringdown analysis with PyCBC
 [![Open Tutorial 8](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/gwastro/pycbc-tutorials/blob/master/tutorial/inference_8_BHRingdown.ipynb)
 
+Tutorial 9: Adding custom models to PyCBC
```

Comment:
Maybe use a more descriptive phrase than 'model' in the title here, just so it is clear? 

Maybe something like the following? 

Tutorial 9: Adding custom models to PyCBC (e.g. alternate likelihoods, GRB / Kilonova models, etc)

Reply:
Good idea